docs/blog/2018-06-15-why-we-built-the-site-showcase/index.md

```diff
@@ -15,76 +15,76 @@ We recently finished the first phase of the [Site Showcase](https://next.gatsbyj
 
 If you’d like to submit a site to the showcase, please follow the [instructions in this document](https://github.com/gatsbyjs/gatsby/blob/v2/docs/docs/site-showcase-submissions.md).
  
-# Why build a site showcase?
+## Why build a site showcase?
 
-We built the showcase to make it easier for you to find the source code for Gatsby sites and easier to show off what Gatsby can do to potential clients / coworkers.
+We built the showcase to make it easier for you to find the src code for open source Gatsby sites and easier to show off what Gatsby can do to potential clients / coworkers.
 
-## Source code
+### Src code
```

Comment:
Sorry — I think my feedback was unclear. I think we should use "source", not "src".